OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/12.2.0.1/AddNode.sh

```diff
@@ -575,50 +733,50 @@ local vip_hostname=$VIP_HOSTNAME
 local cmd
 local stat
 
+if [ -z $CRS_NODES ]; then
+  CLUSTER_NODES=$PUBLIC_HOSTNAME
+else
+  IFS=', ' read -r -a CLUSTER_NODES <<< "$CRS_NODES"
+fi
+
 if [ -f "$logdir/cluvfy_check.txt" ]; then
 print_message "Moving any exisiting cluvfy $logdir/cluvfy_check.txt to $logdir/cluvfy_check_$TIMESTAMP.txt"
 mv $logdir/cluvfy_check.txt $logdir/cluvfy_check."$(date +%Y%m%d-%H%M%S)".txt
 fi
 
-cmd='su - $GRID_USER -c "ssh $node  \"$GRID_HOME/runcluvfy.sh stage -pre nodeadd -n $hostname -vip $vip_hostname\" | tee -a $logdir/cluvfy_check.txt"'
+#cmd='su - $GRID_USER -c "ssh $node  \"$GRID_HOME/runcluvfy.sh stage -pre nodeadd -n $hostname -vip $vip_hostname\" | tee -a $logdir/cluvfy_check.txt"'
+#eval $cmd
+
+print_message "Nodes in the cluster ${CLUSTER_NODES[@]}"
+for cls_node in "${CLUSTER_NODES[@]}"; do
+print_message "ssh to the node $node and executing cvu checks on $cls_node"
+cmd='su - $GRID_USER -c "ssh $node  \"$GRID_HOME/runcluvfy.sh stage -pre nodeadd -n $cls_node\" | tee -a $logdir/cluvfy_check.txt"'
 eval $cmd
```

Comment:
What's the benefit of setting `cmd` and then immediately `eval`'ing it? Why not just `eval` the string on line 753? This is true for all instances of the `cmd`/`eval` pairs later.